src/components/Book/Book.js

```diff
@@ -1,31 +1,38 @@
 import React from 'react';
 // eslint-disable-next-line import/no-extraneous-dependencies
 import PropType from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { booksActions } from '../../redux/books/booksSlice';
+import Button from '../Button/Button';
+
+/* eslint-disable */
+function Book({ ID, category, title, author }) {
+  /* eslint-enable */
```

Comment:
- [ ] Please remove the `eslint-disable` statements as this defies the purpose of using linters in the first place.